### PR TITLE
Add cocoapods support instruction

### PR DESCRIPTION
**Pull request by raosanat**
## Purpose

* ...

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
Get the voice video calling sample from here
https://github.com/Azure-Samples/communication-services-ios-quickstarts/tree/main/Add%20Voice%20Calling
```

* Test the code

Create Podfile as in quick start sample and follow the instructions there
```
```

## What to Check
Make sure the xcoworkspace is created properly
* ...

### CODE REVIEW

**Comment by ddematheu2** [comment]
@raosanat can we also update docs.microsoft.com (https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/getting-started-with-calling?pivots=platform-ios)  as well as the core ReadMe (https://github.com/Azure-Samples/communication-services-ios-quickstarts)

File: Add Voice Calling/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ To complete this tutorial, you’ll need the following prerequisites:
 - A Mac running [Xcode](https://go.microsoft.com/fwLink/p/?LinkID=266532), along with a valid developer certificate installed into your Keychain.
 - A deployed Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource).
 - A [User Access Token](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-csharp) for your Azure Communication Service.
-- Run the shell script `BuildAzurePackages.sh` which is part of the release to build `AzureCommunication.framework` and `AzureCore.framework` packages and include the 2 packages in the sample. **BuildAzurePackages.sh script works only with Xcode 11.5 and above**
+- Create a `Podfile` for your application to fetch `AzureommunicationCalling.framework`, `AzureCore.framework` and `AzureCommunication.framework` using CocoaPods.
```
Comment (ddematheu2) [inline Add Voice Calling/README.md:13]:
```suggestion
- Create a `Podfile` for your application to fetch `AzureCommunicationCalling.framework`, `AzureCore.framework` and `AzureCommunication.framework` using CocoaPods.
```

**Review by ddematheu2 — APPROVED** [review]

File: Add Voice Calling/README.md (hunk 1 above)
Comment (ddematheu2) [reply Add Voice Calling/README.md:13]: @raosanat validate before commiting suggestion.

**Comment by raosanat** [comment]
> @raosanat can we also update docs.microsoft.com (https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/getting-started-with-calling?pivots=platform-ios) as well as the core ReadMe (https://github.com/Azure-Samples/communication-services-ios-quickstarts)

I have already created PR's for those, waiting for Mick to approve them

https://github.com/MicrosoftDocs/azure-docs/pull/66252
https://github.com/MicrosoftDocs/azure-docs/pull/66251

**Comment by ddematheu2** [comment]
> > @raosanat can we also update docs.microsoft.com (https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/getting-started-with-calling?pivots=platform-ios) as well as the core ReadMe (https://github.com/Azure-Samples/communication-services-ios-quickstarts)
> 
> I have already created PR's for those, waiting for Mick to approve them
> 
> [MicrosoftDocs/azure-docs#66252](https://github.com/MicrosoftDocs/azure-docs/pull/66252)
> [MicrosoftDocs/azure-docs#66251](https://github.com/MicrosoftDocs/azure-docs/pull/66251)

PR for https://github.com/Azure-Samples/communication-services-ios-quickstarts/blob/main/README.md?

**Pull request merged**